src/engine/scenes/TestSceneKevin.cpp

```diff
@@ -65,6 +66,11 @@ namespace lei3d
 		CharacterController* characterController = backpackObj.AddComponent<CharacterController>();
 		characterController->Init(1.f, 3.f);
 
+		TriggerCollider* triggerCollider = backpackObj.AddComponent<TriggerCollider>();
```

Comment:
To make sure I understand::


This is a test trigger collider component that will later be used to develop other derived collider components? (that does different things if anything collides with them?). I.e. I can put this on a flower so if I collide with the flower I can switch to a new level?